uk.co.compucorp.civicrm.hrcontactactionsmenu/scss/components/_contact-actions-menu.scss

```diff
@@ -0,0 +1,70 @@
+.crm_contact-actions {
+  padding: 0;
+  width: 1065px !important;
+  height: auto;
+
+  #bootstrap-theme .row {
+    margin: 0;
+  }
+
+  &::before {
+    background: $gray-light !important;
+    border-bottom: none !important;
+    height: 20px !important;
+    left: 0 !important;  
+    top: -15px !important;
+    width: 93px !important;
+  }
+
+  .crm_contact-actions__header {
+    .crm_contact-actions__user-info {
+      padding: 5px 12px 12px !important;
```

Comment:
`padding` is not a layout rule, so there should not need to use the parent, otherwise what's the point of using BEM

This:
```scss
.crm_contact-actions__header {
  .crm_contact-actions__user-info {
    padding: 5px 12px 12px !important;
  }
}
```

should be this

```scss
.crm_contact-actions__user-info {
  padding: 5px 12px 12px !important;
}
```